deployment/common/changeset/deploy_link_token.go

```diff
@@ -110,10 +114,25 @@ func deployLinkTokenContractEVM(
 ) (*deployment.ContractDeploy[*link_token.LinkToken], error) {
 	linkToken, err := deployment.DeployContract[*link_token.LinkToken](lggr, chain, ab,
 		func(chain deployment.Chain) deployment.ContractDeploy[*link_token.LinkToken] {
-			linkTokenAddr, tx, linkToken, err2 := link_token.DeployLinkToken(
-				chain.DeployerKey,
-				chain.Client,
+			var (
+				linkTokenAddr common.Address
+				tx            *eth_types.Transaction
+				linkToken     *link_token.LinkToken
+				err2          error
 			)
+			if !chain.IsZk {
```

Comment:
Ah I see that this `IsZk` check is needed here, for some reason I thought we were getting this info from the chain-selectors API. Any reason this isn't in chain-selectors (perhaps a family - `ZKSyncEVM` or something similar)

Reply:
We are flagging the env for simplicity, we only use this flag in `chainlink/deployment`. We will set it up from `chainlink-deployments` for the necessary selectors